File: 0455-assign-cookies/0455-assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.sort()
        i,j=0,0
        ct=0
        while i<len(g) and j<len(s):
            if s[j]>=g[i]:
                ct+=1
                i+=1
            j+=1
        return ct

File: 0455-assign-cookies/0455-assign-cookies.cs
public class Solution {
    public int FindContentChildren(int[] g, int[] s) {
        Array.Sort(g);
        Array.Sort(s);
        int ct_g=0;
        int ct_s=0;
        while(ct_g<g.Length && ct_s<s.Length){
            if(s[ct_s]>=g[ct_g]){
                ct_g++;
            }
            ct_s++; 
        }
        return ct_g;
        
    }
}